src/components/todoHome.vue

```diff
@@ -1,24 +1,34 @@
 <template>
-  <div class="home-page">
-    <div class="home-page__card">
-      <h2 class="home-page__card-title">Get Things Done!</h2>
-      <div class="home-page__card-button-wrapper">
-        <v-btn class="home-page__card-button" to="/task">+ New Task</v-btn>
-      </div>
+  <div class="home-page__card">
+    <h2 class="home-page__card-title">Get Things Done!</h2>
+
```

Comment:
Please remove all the unwanted spaces from the entire file. Please make sure that you are not repeating mistakes.